src/bigscreenplayer.test.js

```diff
@@ -786,14 +786,12 @@ describe("Bigscreen Player", () => {
       initialiseBigscreenPlayer({ windowType: WindowTypes.SLIDING })
 
       const callback = jest.fn()
-      const endOfStreamWindow = bigscreenPlayerData.time.windowEndTime - 2
+      const endOfStreamWindow = 100 - 2
```

Comment:
We use `100` in a few locations in this file, can we make it a const, so if it changes for whatever reason in the future, you don't need to update it here, L793, and L820

Reply:
IMO that's optimising for the current state of the tests and may make it harder to change in the future. I tend to make my tests as unoptimised as possible because find/replace can do the job